### ISSUE TITLE
[transformed AMP]  Validator does not support blurry image placeholders 

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amphtml#22151]
Here is a blurry image placeholder generated by AMP Optimizer:

```
<amp-img src="https://amp.dev/static/samples/img/amp.jpg" width="1080" height="610" layout="responsive" 
         class="i-amphtml-layout-responsive i-amphtml-layout-size-defined" 
         i-amphtml-layout="responsive" noloading="">
  <i-amphtml-sizer style="display:block;padding-top:56.4815%;"></i-amphtml-sizer>
  <img class="i-amphtml-blurry-placeholder" 
       placeholder="" 
       src="data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http%3A//www.w3.org/2000/svg' xmlns%3Axlink='http%3A//www.w3.org/1999/xlink' viewBox='0 0 10 6'%3E%3Cfilter id='b' color-interpolation-filters='sRGB'%3E%3CfeGaussianBlur stdDeviation='.5'%3E%3C/feGaussianBlur%3E%3CfeComponentTransfer%3E%3CfeFuncA type='discrete' tableValues='1 1'%3E%3C/feFuncA%3E%3C/feComponentTransfer%3E%3C/filter%3E%3Cimage filter='url(%23b)' x='0' y='0' height='100%25' width='100%25' xlink%3Ahref='data%3Aimage/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAGCAYAAAD68A/GAAAA+ElEQVR4AQXBPS8DYQDA8f9z99xLm9x5uUQYDIaWayMq6WSRIgZB8AEaX0CMEkEMEt/B5BMwMDQ1dPQ6WEikvVWlSBon0d6r30/sVafS89ogSZJQEIJvVSVjmrhJzP1fj04QEgR9lPqDzuH+KpZt09J05jfLlAwNpzxBV8DSoks+N4bUrHEmcw5u3uH2zmN3Z4EzU1KpuDS8NlsbJX66vygry0U+v3zW12YxTIOnR4/t6hzNZhvf7zE6YnPTeEOdLswcn5xe4wzbtLwO2azOwdEFtpXh5fWDMIy4vHpGmLqbqkNFjPCdNO4jpSSKYlBUoigglQPEicE/wDBT/+m3qawAAAAASUVORK5CYII='%3E%3C/image%3E%3C/svg%3E" alt="">
</amp-img>
```

Which will result in the following validation error:

```
The tag 'img' may only appear as a descendant of tag 'noscript'. Did you mean 'amp-img'
```

**Comment by sebastianbenz** [comment]
//cc @honeybadgerdontcare

**Comment by choumx** [comment]
Are the other SSR/Optimizer transformations valid yet? This bug should be categorized under that same effort.

**Comment by sebastianbenz** [comment]
All other SSR transformations should be supported. Here is a live version of amp.dev publishing valid transformed AMP: https://amp-dev-staging.appspot.com (this might go back to the normal version soon).

See also https://github.com/ampproject/amphtml/issues/21941.

**Comment by honeybadgerdontcare** [comment]
I don't believe this is a supported transformation. AFAIK we don't do it in the AMP Cache and it's not listed under the SSR documentation.

**Comment by choumx** [comment]
It's a WIP transformation. `go/amp-blurry-image` for context.

**Comment by cramforce** [comment]
This has shipped on the AMP Cache. I think that in general, whether an optimization is implemented on the AMP Cache should not block that it can be valid.

**Comment by Gregable** [comment]
I think the real problem might be that the inserted image src is actually an SVG. So, to do this, we'd need to decode the url-encoded SVG, and then run an SVG parser to ensure that the contents of this string really are a blurry image, rather than something with an http network leak.

I've only briefly guessed at the requirements a bit, so I could be off.

**Comment by Gregable** [comment]
@sebastianbenz has this been resolved fully (ie: per https://github.com/ampproject/amphtml/pull/27518)?

**Comment by sebastianbenz** [comment]
No, blurry placeholders are still not valid AMP, e.g. `i-amphtml-blurry-placeholder` and `src="data:image/svg+xm...` cause validation errors.

**Comment by honeybadgerdontcare** [comment]
This is fixed in cl/338138658
